client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -99,6 +99,22 @@ const Row = styled.div< { isHiddenInMobile?: boolean } >`
 	` ) }
 `;
 
+const PlanRow = styled( Row )`
+	&:last-of-type {
+		display: none;
+	}
+
+	${ plansBreakSmall( css`
+		border-bottom: none;
+
+		&:last-of-type {
+			display: flex;
+			padding-top: 0;
+			padding-bottom: 24px;
```

Comment:
We'll probably need to move the `padding-bottom: 24px;` to `.plan-comparison-grid__header.plan-is-footer`, otherwise the border stops before the bottom.
<img width="252" alt="Screenshot 2023-02-09 at 13 23 15" src="https://user-images.githubusercontent.com/2749938/217799529-d60387b9-660d-4f33-ac43-cded3f654b0e.png">